packages/catalog-process/src/services/readModelService.ts

```diff
@@ -33,7 +34,7 @@ import { ApiGetEServicesFilters } from "../model/types.js";
 
 async function getEService(
   eservices: EServiceCollection,
-  filter: Filter<WithId<WithMetadata<EService>>>
+  filter: Filter<WithId<WithMetadata<EServiceReadModel>>>
```

Comment:
Is this required? Can we convert to `EService` before returning?

So that we work only with `EService` here in the `catalog-process`, and not with `EServiceReadModel`

Reply:
My bad, this is the type of `filter`, not the return type - ignore the question :))